WordPress/Classes/ViewRelated/Site Creation/Final Assembly/SiteAssemblyWizardContent.swift

```diff
@@ -309,6 +313,20 @@ extension SiteAssemblyWizardContent: NUXButtonViewControllerDelegate {
             guard let self = self else {
                 return
             }
+            if AppConfiguration.isWordPress && Blog.count(in: ContextManager.sharedInstance().mainContext) == 1 {
+                let viewModel = JetpackFullscreenOverlayGeneralViewModel(
+                    phase: .four,
+                    source: .phaseFourOverlay,
+                    blog: blog,
+                    coordinator: JetpackDefaultOverlayCoordinator()
+                )
+                let overlayViewController = JetpackFullscreenOverlayViewController(with: viewModel)
+                let navigationViewController = UINavigationController(rootViewController: overlayViewController)
+                JetpackFeaturesRemovalCoordinator.presentOverlay(
```

Comment:
I don't think there's a need to manually trigger an overlay.
Correctly configuring `MySiteViewController.willDisplayPostSignupFlow` before triggering the site creation flow should stop the overlay from being presented at the wrong time and should then automatically be displayed after the site is created.